crates/core/tedge_mapper/src/core/mapper.rs

```diff
@@ -19,17 +20,17 @@ pub async fn create_mapper(
     info!("{} starting", app_name);
 
     let mapper_config = converter.get_mapper_config();
-    let mqtt_client = Connection::new(&mqtt_config(
-        app_name,
-        &mqtt_host,
-        mqtt_port,
-        mapper_config.in_topic_filter.clone(),
-    )?)
-    .await?;
+    let mut topics = mapper_config.in_topic_filter.clone();
```

Comment:
Actually, they are topic filters.
```suggestion
    let mut topic_filters = mapper_config.in_topic_filter.clone();
```